File: mobile/src/main/java/tr/xip/prayertimes/db/table/PrayerTimesTable.java
package tr.xip.prayertimes.db.table;

import android.provider.BaseColumns;

public class PrayerTimesTable implements BaseColumns {
    public static final String TABLE_NAME = "prayer_times";
    public static final String COLUMN_NAME_ID = "_id";
    public static final String COLUMN_NAME_LOCATION = "location";
    public static final String COLUMN_NAME_DATE = "date";
    public static final String COLUMN_NAME_FAJR = "fajr";
    public static final String COLUMN_NAME_SUNRISE = "sunrise";
    public static final String COLUMN_NAME_DHUHR = "dhuhr";
    public static final String COLUMN_NAME_ASR = "asr";
    public static final String COLUMN_NAME_MAGHRIB = "maghrib";
    public static final String COLUMN_NAME_ISHA = "isha";
    public static final String COLUMN_NULLABLE = "nullable";
}

File: mobile/src/main/kotlin/tr/xip/prayertimes/db/table/PrayerTimesTable.kt
package tr.xip.prayertimes.db.table

import android.provider.BaseColumns

object PrayerTimesTable : BaseColumns {
    val TABLE_NAME = "prayer_times"
    val COLUMN_NAME_ID = "_id"
    val COLUMN_NAME_LOCATION = "location"
    val COLUMN_NAME_DATE = "date"
    val COLUMN_NAME_FAJR = "fajr"
    val COLUMN_NAME_SUNRISE = "sunrise"
    val COLUMN_NAME_DHUHR = "dhuhr"
    val COLUMN_NAME_ASR = "asr"
    val COLUMN_NAME_MAGHRIB = "maghrib"
    val COLUMN_NAME_ISHA = "isha"
    val COLUMN_NULLABLE = "nullable"
}
